spring-cloud-sleuth-otel-autoconfigure/src/main/java/org/springframework/cloud/sleuth/autoconfig/otel/zipkin2/ZipkinOtelAutoConfiguration.java

```diff
@@ -65,13 +64,7 @@
 		ZipkinSpanExporter otelZipkinSpanExporter(ZipkinProperties zipkinProperties,
 				@Qualifier(ZipkinAutoConfiguration.SENDER_BEAN_NAME) Sender sender, Environment env) {
 			return ZipkinSpanExporter.builder().setEndpoint(zipkinProperties.getBaseUrl() + "api/v2/spans")
-					.setSender(sender).setEncoder(zipkinProperties.getEncoder())
-					.setServiceName(
-							StringUtils.hasText(zipkinProperties.getService().getName())
-									? zipkinProperties.getService().getName()
-									: env.getProperty("spring.application.name", env.getProperty(
-											"spring.zipkin.service.name", ZipkinSpanExporter.DEFAULT_SERVICE_NAME)))
-					.build();
+					.setSender(sender).setEncoder(zipkinProperties.getEncoder()).build();
```

Comment:
So in this autoconfig I'd create this `ResourceCustomizer`. Will it be common for the users to customize the resources?

Reply:
I think it will be. This definitely seems like a good enhancement to add to this project.